main.go

```diff
@@ -1,8 +1,9 @@
 package main
 
 import (
-	"./requestor"
 	"flag"
+
+	"github.com/t0xic0der/prudence/requestor"
```

Comment:
You would want to restore it back to the way it was before as `requestor` is packaged within `prudence` and there is no need to fetch it like this.